test/authz-filler/main.go

```diff
@@ -0,0 +1,96 @@
+// A quick way to fill up a database with a large number of authz objects, in
```

Comment:
We should probably use this in integration tests to test the purger, can you open a ticket for that.

Reply:
We already test the purger in the integration tests using the "real" system to create authzs. This is intended for load testing, which takes too long to be practical in the integration test.